extensions/vscode/README.md

```diff
@@ -69,6 +69,18 @@ See the
 [Updating Previously Deployed Content](https://github.com/posit-dev/publisher/blob/main/docs/vscode.md#updating-previously-deployed-content)
 documentation for more information.
 
+### Seemless collaboration
+
+Collaborate easily with your team by sharing your configurations and
+deployments. Others can pick up where you left off and
+
+Allow collaborators to easily pick up where you left off by using the `.posit`
```

Comment:
Should we switch this to call out the `.posit/publisher` files rather than just the `.posit` files?

Reply:
Yes that is a great point. I'll get that updated.